src/main/java/com/adobe/commerce/cif/model/cart/Cart.java

```diff
@@ -35,20 +35,20 @@
     @ApiModelProperty(value = "If set, this defines the customer owning this cart. If not set, the cart is an anonymous cart.")
     protected String customerId;
     
-    @ApiModelProperty(value = "The net total price for the cart, including discounts, and shipping, but excluding any taxes.")
+    @ApiModelProperty(value = "The net total price for the cart, including discounts, and shipping, but excluding any taxes.", required = true)
     protected Price netTotalPrice;
     
-    @ApiModelProperty(value = "The gross total price for the cart, including discounts, shipping, and all taxes.")
+    @ApiModelProperty(value = "The gross total price for the cart, including discounts, shipping, and all taxes.", required = true)
     protected Price grossTotalPrice;
     
     @ApiModelProperty(value = "The product subtotal for the cart, including discounts and with or without taxes depending if the product prices include taxes or not." +
-                              "Until a shipping address is set, this field is typically used as the temporary cart total until it is known if prices include taxes or not.")
+                              "Until a shipping address is set, this field is typically used as the temporary cart total until it is known if prices include taxes or not.", required = true)
     protected Price totalProductPrice;
 
-    @ApiModelProperty(value = "The cart tax info, including cart entries tax and shipping info tax.")
+    @ApiModelProperty(value = "The cart tax info, including cart entries tax and shipping info tax.", required = true)
```

Comment:
If the shipping info is not defined this will be empty.

Reply:
Then it shouldn't be required and description needs to be updated.

Reply:
Done.